packages/components/package.json

```diff
@@ -80,6 +80,7 @@
     "@shoelace-style/animations": "^1.1.0",
     "@shoelace-style/localize": "^3.1.2",
     "classix": "^2.1.35",
+    "countup.js": "^2.8.0",
```

Comment:
Do not add this to dependencies, this is only for documentation and is therefore a devDependency.